src/patternfly/components/Skeleton/skeleton.scss

```diff
@@ -0,0 +1,217 @@
+.pf-c-skeleton {
+  // Padding modifier
+  --pf-c-skeleton--m-padding--Padding: var(--pf-global--spacer--lg);
+
+  // Element
+  --pf-c-skeleton__element--BackgroundColor: var(--pf-global--palette--black-150);
+  --pf-c-skeleton__element--Height: var(--pf-global--spacer--lg);
```

Comment:
Should this just be the current font size * line height maybe? I think that would make a good default.

Reply:
Yep and that aligns with the text modifiers - sounds good